package.json

```diff
@@ -37,32 +50,39 @@
     "decanter": "git://github.com/su-sws/decanter.git#v7",
     "react-hero-icon": "^0.0.7"
   },
+  "peerDependencies": {
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0"
+  },
   "devDependencies": {
     "@babel/core": "^7.12.16",
-    "@babel/preset-env": "^7.12.16",
-    "@storybook/addon-a11y": "^6.1.17",
-    "@storybook/addon-actions": "^6.1.15",
-    "@storybook/addon-essentials": "^6.1.15",
+    "@babel/plugin-transform-react-jsx": "^7.12.17",
+    "@babel/preset-env": "^7.12.17",
+    "@babel/preset-react": "^7.12.13",
+    "@storybook/addon-a11y": "^6.1.19",
+    "@storybook/addon-actions": "^6.1.19",
+    "@storybook/addon-essentials": "^6.1.19",
     "@storybook/addon-links": "^6.1.15",
-    "@storybook/addon-viewport": "^6.1.17",
-    "@storybook/node-logger": "^6.1.15",
-    "@storybook/react": "^6.1.15",
+    "@storybook/addon-viewport": "^6.1.19",
+    "@storybook/node-logger": "^6.1.19",
+    "@storybook/react": "^6.1.19",
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.5",
     "@whitespace/storybook-addon-html": "^4.2.0",
     "babel-jest": "^26.6.3",
-    "classnames": "^2.2.6",
```

Comment:
Uh oh, does that mean no dedupe? 👀

Reply:
https://www.npmjs.com/package/clsx#usage

dedupe is the default behavior of this package. It is more modern and quicker than classnames. Also, https://github.com/JedWatson/classnames/issues/220

Reply:
Yeah pretty cool. I'm checking it out. It's really popular too. Nice find.

Reply:
Ah, disabled button needs to be re-worked... arg.